src/main/java/org/ohdsi/circe/cohortdefinition/VisitDetail.java

```diff
@@ -41,44 +41,44 @@ public class VisitDetail extends Criteria {
   @JsonProperty("VisitDetailEndDate")
   public DateRange visitDetailEndDate;
 
-  @JsonProperty("VisitDetailType")
-  public Concept[] visitDetailType;
+  @JsonProperty("VisitDetailTypeCS")
+  public Integer visitDetailTypeCS;
 
   @JsonProperty("VisitDetailTypeExclude")
   public boolean visitDetailTypeExclude = false;
 
-  @JsonProperty("VisitDetailSourceConcept")
-  public Integer visitDetailSourceConcept;
+  @JsonProperty("VisitDetailSourceConceptCS")
+  public Integer visitDetailSourceConceptCS;
```

Comment:
Revert these, please.

Reply:
Do you mean reverting the type of this field to Concept[] too?

Reply:
This field (referencing a soruc econcept) was never a Concept[].  When I said revert, I meant change the name from `VisitDetailSourceConceptCS` back to `VisitDetailSourceConcept`.   It was always an Integer and it should remain an Integer.

Reply:
fixed